src/utils/shadowRoot.ts

```diff
@@ -0,0 +1,10 @@
+export const getShadowRoot = (node: any) => {
```

Comment:
Can we type it somehow? Would the code below work?

```suggestion
export const getShadowRoot = (node: HTMLElement) => {
```